client/src/components/LoginSession/LoginSession.js

```diff
@@ -5,6 +5,8 @@ import moment from 'moment';
 import confirm from '@silverstripe/reactstrap-confirm';
 import Button from 'components/Button/Button';
 
+const format = 'L LT';
```

Comment:
For the mere mortals who have yet to memorise all possible time format strings.

```suggestion
/**
 * Local date and Local time format. e.g.: `04/15/2021 1:31 PM` for en_US
 * @type {string}
 */
const format = 'L LT';
```

Reply:
Updated